src/module-elasticsuite-core/Model/Healthcheck/HyvaCompatibilityCheck.php

```diff
@@ -58,7 +59,7 @@ public function __construct(
         array $packagesToCheck = [],
         int $sortOrder = 40
     ) {
-        parent::__construct($urlBuilder, $sortOrder);
+        parent::__construct($urlBuilder, $sortOrder, MessageInterface::SEVERITY_MAJOR);
```

Comment:
Please add the severity is as an exposed constructor parameter, after the sortOrder.